deployment/submit_emr_job_train.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+This script spawns a spark emr cluster on AWS and submits a job to run the given src code.
+
+Copyright © 2018 Red Hat Inc.
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+import logging
+import boto3
+from time import gmtime, strftime
+import daiquiri
+import os
+
+daiquiri.setup(level=logging.DEBUG)
+_logger = daiquiri.getLogger(__name__)
+
+COMPONENT_PREFIX = "go-cve"
+AWS_S3_ACCESS_KEY_ID = os.environ.get('AWS_S3_ACCESS_KEY_ID', '')
+AWS_S3_SECRET_ACCESS_KEY = os.environ.get('AWS_S3_SECRET_ACCESS_KEY', '')
+_logger.info(AWS_S3_ACCESS_KEY_ID)
+_logger.info(AWS_S3_SECRET_ACCESS_KEY)
+
+def submit_job():
+    """Submit new job with specified parameters."""
+    str_cur_time = strftime("%Y_%m_%d_%H_%M_%S", gmtime())
+    # S3 bucket/key, where the input spark job ( src code ) will be uploaded
+    s3_bucket = os.environ.get('AWS_S3_BUCKET_NAME', 'avgupta-dev-emr-jobs')
```

Comment:
can we read all required configuration form cloud constants

Reply:
These are EMR specific variables that cannot be referenced directly in the source code. They're strictly for the deployment script.